Makefile

```diff
@@ -3,7 +3,7 @@
 
 SHELL = /usr/bin/env bash
 
-ifneq (${TARGET}, prod)
+ifndef TARGET
 TARGET = dev
 endif
```

Comment:
Slight difference in that `TARGET=pord` probably breaks badly rather than defaulting to `dev`. Neither's ideal, better would be an error if it's not in the set list. Wonder if we can extract a change here to cater for that.